lib/improver/wind_downscaling.py

```diff
@@ -552,7 +546,7 @@ def _delta_height(self):
                 height difference, ppgrid-model
 
         """
-        delt_z = np.ones(self.pporo.shape) * RMDI
+        delt_z = np.ones(self.pporo.shape)
```

Comment:
delt_z is set to 1 where not updated by the mask below. Is this right?

Reply:
This is one of Katie's code changes - I'm not too sure. Looking into it.

Reply:
Sorted